components/HelpModal.go

```diff
@@ -0,0 +1,77 @@
+package components
+
+import (
+	"github.com/gdamore/tcell/v2"
+	app "github.com/jorgerojas26/lazysql/app"
+	"github.com/jorgerojas26/lazysql/commands"
+	. "github.com/jorgerojas26/lazysql/keymap"
+	"github.com/rivo/tview"
+)
+
+type HelpModal struct {
+	*tview.Flex
+}
+
+func NewHelpModal() *HelpModal {
+
+	// Colors
+	BorderColor := tcell.ColorGreen
+	SelectedColor := tcell.ColorBlue
```

Comment:
I would expect this to be constants defined in the package, not in the function.

Also, a shared prefix make more sense, as in tcell, so I would have used

```suggestion
	colorBorder := tcell.ColorGreen
	colorSelected := tcell.ColorBlue
```

Also, it's uncommon to define variables that start with an uppercase when they are not intended to be exported